react/components/SearchQuery.js

```diff
@@ -164,6 +164,16 @@ const useQueries = (variables, facetsArgs) => {
     },
   })
 
+  const initialMap = () => {
```

Comment:
I would suggest to change the name. `initialMap` looks like an object instead of a function.